keps/sig-cloud-provider/0013-build-deploy-ccm.md

```diff
@@ -0,0 +1,496 @@
+---
+kep-number: 13
+title: Switching To Cloud Provider Repo And Builds
+authors:
+  - "@cheftako"
+  - "@calebamiles"
+  - "@nckturner"
+owning-sig: sig-apimachinery
+participating-sigs:
+  - sig-apps
+  - sig-aws
+  - sig-azure
+  - sig-cloud-provider
+  - sig-gcp
+  - sig-network
+  - sig-openstack
+  - sig-storage
+reviewers:
+  - "@andrewsykim"
+  - "@calebamiles"
+  - "@nckturner"
+approvers:
+  - "@thockin"
+editor: TBD
+status: provisional
+---
+
+# Switching To Cloud Provider Repo And Builds 
+
+## How To Remove Cloud Provider Code From Kubernetes Core
+
+## Table of Contents
+
+- [Switching To Cloud Provider Repo And Builds](#switching-to-cloud-provider-repo-and-builds)
+   - [Table of Contents](#table-of-contents)
+   - [Terms](#terms)
+   - [Summary](#summary)
+   - [Motivation](#motivation)
+      - [Goals](#goals)
+      - [Intermediary Goals](#intermediary-goals)
+      - [Non-Goals](#non-goals)
+   - [Proposal](#proposal)
+   - [Building CCM For Cloud Providers](#building-ccm-for-cloud-providers)
+      - [Background](#background)
+      - [Design Options](#design-options)
+          - [Staging](#staging)
+          - [Importing the Cloud Provider Implementation](#importing-the-cloud-provider-implementation)
+          - [Build Targets](#build-targets)
+          - [Releases](#releases)
+          - [Migrating to a CCM Build](#migrating-to-a-ccm-build)
+          - [Flags, Service Accounts, etc](#flags,-service-accounts,-etc)
+          - [Deployment Scripts](#deployment-scripts)
+          - [KubeAdm](#kubeadm)
```

Comment:
`KubeAdm` should be lowercase `kubeadm` everywhere in the doc.

Reply:
Fixed